src/handlers/appInfoHandlers.ts

```diff
@@ -17,6 +19,23 @@ export class AppInfoHandlers {
       return app.getVersion();
     });
 
+    // eslint-disable-next-line @typescript-eslint/require-await
+    ipcMain.handle(IPC_CHANNELS.GET_BASE_PATH, async (): Promise<string | null | undefined> => {
+      return useDesktopConfig().get('basePath');
+    });
+
+    ipcMain.handle(IPC_CHANNELS.SET_BASE_PATH, async (): Promise<boolean> => {
+      const currentBasePath = useDesktopConfig().get('basePath');
+
+      const result = await appWindow.showOpenDialog({ properties: ['openDirectory'], defaultPath: currentBasePath });
+      if (result.canceled || !(result.filePaths.length > 0)) return false;
+
+      const basePath = result.filePaths[0];
+      useDesktopConfig().set('basePath', basePath);
+      // TODO: Replace with new base path config
```

Comment:
Can you file an issue tracking this TODO, and add the issue URL here? The issue should explain what needs to be done in detail.